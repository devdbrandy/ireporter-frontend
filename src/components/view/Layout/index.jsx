import React from 'react';
import Header from '../Header';
// import Footer from '../Footer';

const Layout = ({ children }) => (
  <>
    <Header />
    {children}
    {/* TODO: fix issue with aligning <Footer /> */}
  </>
);

export default Layout;
