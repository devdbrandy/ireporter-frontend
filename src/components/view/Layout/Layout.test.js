import React from 'react';
import { shallow } from 'enzyme';
import Layout from './index';

/**
 * Wrapper for enzyme shallow component
 *
 * @returns {object} The mocked props and component
 */
const setup = () => {
  const props = {
    children: {},
  };

  const component = shallow(<Layout {...props} />);
  return { props, component };
};

describe('<Layout />', () => {
  it('renders without crashing given the required props', () => {
    const { component } = setup();
    expect(component).toMatchSnapshot();
  });
});
