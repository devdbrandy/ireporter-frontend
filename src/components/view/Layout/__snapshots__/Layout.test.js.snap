// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders without crashing given the required props 1`] = `
<Fragment>
  <Header />
  <Component />
</Fragment>
`;
